scripts/utils/exec.ts

```diff
@@ -1,10 +1,12 @@
-require('dotenv').config();
-const { time } = require('@openzeppelin/test-helpers');
-const hre = require('hardhat');
+import hre, { ethers, artifacts } from 'hardhat';
+import { time } from '@openzeppelin/test-helpers';
 
-const GovernorAlpha = artifacts.require('GovernorAlpha');
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+const GovernorAlpha = artifacts.readArtifactSync('GovernorAlpha');
```

Comment:
is this used?

Reply:
We should have a linter that runs locally and in CI that catches unused vars.

Reply:
Nope, removed.

Reply:
@sol-sr absolutely we should. That's in the pipeline